tests/test_run.py

```diff
@@ -17,10 +17,10 @@ def test_run_broken_command_returns_nonzero():
     assert output == b""
     assert error != b""
     assert code != 0
-
-
 # pylint: disable=unused-argument
 # pylint: disable=redefined-outer-name
+
```

Comment:
I'm confused as to why this would not have failed in linting before...

Reply:
I'm still trying to get my IDE setup correctly it doesn't lint well when I'm switching between projects.